docs/room/realtimemodel-primitives.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: docs
+title: RealtimeModel Primitives
+---
+# RealtimeModel Primitives
+
+In [RealtimeModel](./realtimemodel.md) definitions the `[RealtimeProperty]` attribute can be used on properties of many different types. Here's the full list:
+
+# C# primitives
+* `bool`
+* `byte`
+* `sbyte`
+* `short`
+* `ushort`
+* `int`
+* `uint`
+* `float`
+* `double`
+* `string`
+
+# Unity primitives
+* `Color`
+* `Vector2`
+* `Vector3`
+* `Vector4`
+* `Quaternion`
+
+# Collections
+Classes that implement the `ICollection` interface (in the `Normal.Realtime.Serialization` namespace) are also supported. These include:
```

Comment:
I think we can leave out the `ICollection` note and just list the supported collections. We don't expect people to implement their own collections and `ICollection` would be better as internal.

I also think we should have a note here that explicitly says `List`, `Dictionary`, etc are not allowed. Something that quickly explains why people need to use our wrappers, like "our collection classes implement change tracking to create efficient network updates" or whatever.

Reply:
+1